travis/build_and_run_all_spark-3.0_hadoop-3.2.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
:nit script name, use `_`? More tidy and consistent.

Reply:
@gczsjdy  it's done.